packages/sdk/src/sdk/sdk.interfaces.ts

```diff
@@ -122,9 +123,11 @@ export interface WithdrawTransferData<
   destinationBalance: bigint;
   destinationFee: bigint;
   existentialDeposit: bigint;
-  min: bigint;
+  min: XcmFeeWithDecimals<Symbols>;
+  minXcmFeeAsset: XcmFeeWithDecimals<Symbols>;
   native: AssetWithDecimals<Symbols>;
   origin: MoonChain | Chain<ChainKeys>;
+  originXcmFeeAssetBalanceNotEnough: boolean;
```

Comment:
Do we still need this?

Reply:
yes, we use that in the dapp to let the user know if he doesn't have enough NEER for paying the fees. We could also return the origin balance and do the calculations in the dapp

Reply:
I would prefer to avoid changing the interface for now. If we can do calculations in the dapp it would be better for now.

Reply:
Ok but we would still have to change the interface, to return the origin balance

Reply:
ok